plugins/inputs/ah_wireless/ah_wireless.go

```diff
@@ -2935,7 +2962,33 @@ func Send_DeviceStats(t *Ah_wireless, acc telegraf.Accumulator) error {
 		}
 
 		acc.AddGauge("DeviceStats", fields, nil)
+
+		var s string
+
+		s = "-----------------------------------------------\n\n"
+
+		for k, v := range fields {
```

Comment:
This block is practically identical to block above (starting line 2861). Why  not make this a function?